index.html

```diff
@@ -0,0 +1,159 @@
+          </a>
+          <ul class="menu_items flex-row">
+            <li class="nav_items"><a href="#">Home</a></li>
+            <li class="nav_items"><a href="about.html">About</a></li>
+            <li class="nav_items"><a href="">Program</a></li>
+            <li class="nav_items"><a href="">Join</a></li>
+            <li class="nav_items"><a href="">Sponsor</a></li>
+          </ul>
+        </nav>
+      </header>
+      <div class="main-headline">
+        <h2 class="welcome">"Welcome Humans"</h2>
+        <h3 class="welcome-2"></h3>
+        <h1 class="Meco">World is moving with MECO</h1>
+        <h3 class="bottom">
+          We are glad to share you that A joyful celebration believing in the
+          value of Machine Learning and A.I., creating a positive change with
+          people from over 80 countries is taking place in October, in Korea.
+        </h3>
+        <h2 class="date">2015.10.15(THU) ~ 16(FRI)</h2>
+        <h3 class="venue">@ Dhaka, Bangladesh
+        </h5>
+      </div>
+    </section>
+    <section class="program">
+        <h2 class="section-heading">Main Program</h2>
+            <div class="red-bar"></div>
+            <div class=" main-prog-items">
+                <div class="prog-items">
+                    <img src="./Images/program1.png" alt="icon">
+                    <h2>Lecture&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</h2>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, velit?</p>     
+                </div>
+                <div class="prog-items">
+                    <img src="./Images/program2.png" alt="icon">
+                    <h2>CC Exhibition</h2>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, velit?</p>     
+                </div>
+                <div class="prog-items">
+                    <img src="./Images/program3.png" alt="icon">
+                    <h2>Forum&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</h2>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, velit?</p>     
+                </div>
+                <div class="prog-items">
+                    <img src="./Images/program4.png" alt="icon">
+                    <h2>Workshop</h2>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, velit?</p>     
+                </div>
+                <div class="prog-items">
+                    <img src="./Images/program5.png" alt="icon">
+                    <h2>CC Lingration</h2>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, velit?</p>     
+                </div>
+            </div>
+            <button type="submit" class="register-btn">Register Now</button>
+            <a href="#" id="whole-program">See The Whole Program</a>
+    </section>
+    <section class="speaker_section" id="speaker_section">
```

Comment:
- [ ] You did a very great job to make sure the unnecessary space is removed. However, we are still striving to ensure that we match the original design. Through the application of changes the gap on the left and right of the speakers section was removed. Please ensure to apply the gap on the stated sides so that you meet the original design.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/233766527-bb56f85c-2296-443e-bba7-06aa92d85559.png)

Original design 😎 
![image](https://user-images.githubusercontent.com/99394374/233766546-da8cc2be-d84e-4cd6-b655-7cebeec115b3.png)


Design at the previous review point
![image](https://user-images.githubusercontent.com/99394374/233766502-109fd6d2-3795-403d-bb7d-e1330b302e40.png)